converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/Patient/IPatientPropertySynchronizer.cs

```diff
@@ -20,5 +20,10 @@ public interface IPatientPropertySynchronizer
         /// <param name="patient">The <see cref="Patient"/> resource.</param>
         /// <param name="isNewPatient">Flag to determine whether or not the patient being synchronized is new.</param>
         void Synchronize(DicomDataset dataset, Patient patient, bool isNewPatient);
+
+        /// <summary>
+        /// Returns if the property is a required property for the <see cref="Patient"/> resource
+        /// </summary>
+        bool IsRequired();
```

Comment:
Can you add a little details on how this property is used? I see all the properties are marked false

Reply:
I did it based on the cardinality from this link: https://www.hl7.org/fhir/patient.html.  

I used it in the patientsynchronizer where we loop through all the properties. In the case that a property is not required then we we just store to table storage and try to sync the rest of the properties.

Looking further into the spec thou, it actually seems like all of the elements for patient resource are not required so I can probably remove this.

Reply:
Lets remove it, only required is patientId I think

Reply:
removed